class/defaults.yml

```diff
@@ -867,6 +867,7 @@ parameters:
           restoreSA: nextcloudserviceaccount
           restoreRoleRules: ${appcat:defaultRestoreRoleRules}
           additionalInputs:
+            collabora_image: "collabora/code:24.04.8.1.1"
```

Comment:
Please don't hardcode the image here, do it the same way we do it for all other images.